packages/components/src/InputTime/InputTimeSafari.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useLayoutEffect } from "react";
+import TimePolyfill from "time-input-polyfill";
+// eslint-disable-next-line import/no-internal-modules
+import debounce from "lodash/debounce";
```

Comment:
my editor seems to get angry at this 
![image](https://user-images.githubusercontent.com/15986172/63049285-8e668a00-be95-11e9-9e32-851fa3a29a4d.png)

Reply:
Did you do a `npm ci` when you checked this out?

Reply:
yah, `ci` fixed it